include/libc/math.h

```diff
@@ -5,24 +5,37 @@
 
 #define M_PI 3.14159265358979323846f
 #define M_SQRT2 1.41421356237309504880f
-#define FLT_MAX 340282346638528859811704183484516925440.0f
+#define MAXFLOAT 3.40282347e+38f
 #define SHT_MAX 32767.0f
```

Comment:
is this a known original name or a standard name for this? 
am fine with it if so, but it just goes against our convention for similar things

Reply:
This is the standard C name for the largest representable float define in `math.h`.
`FLT_MAX` is another standard C name, but is part of `float.h` instead, which we don't currently have. It might be nice to add additional standard headers in the future, but there's some questions about exactly how much to add so I opted not to add too many new headers for now.

Reply:
ISO C has `FLT_MAX` but not `MAXFLOAT`
https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3096.pdf

`MAXFLOAT` seems to be POSIX
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/math.h.html

Reply:
Specifically ISO C has `FLT_MAX` in `float.h`, which is not a header we currently have. I think it may be nice to add `float.h` and `limits.h` at some point, or I can attempt to add them now if it's much desired?

Reply:
I'm fine with the way it is now personally